packages/framer-motion/src/render/VisualElement.ts

```diff
@@ -827,7 +827,8 @@ export abstract class VisualElement<
     readValue(key: string) {
         return this.latestValues[key] !== undefined || !this.current
             ? this.latestValues[key]
-            : this.readValueFromInstance(this.current, key, this.options)
+            : this.getBaseTargetFromProps(this.props, key) ||
+                  this.readValueFromInstance(this.current, key, this.options)
```

Comment:
Should this be a nullish check instead? In case the returned value is an empty string or 0.

Reply:
Yeah great catch